apps/src/templates/GameButtons.jsx

```diff
@@ -1,48 +1,107 @@
-var msg = require('../locale');
+import msg from '../locale';
 
-var ProtectedStatefulDiv = require('./ProtectedStatefulDiv');
-var commonStyles = require('../commonStyles');
+import ProtectedStatefulDiv from './ProtectedStatefulDiv';
+import commonStyles from '../commonStyles';
+import experiments from '../experiments';
+import classNames from 'classnames';
+import Radium from 'radium';
+import { connect } from 'react-redux';
 
-var styles = {
+const styles = {
   instructionsInTopPane: {
     // common.scss provides an :after selector that ends up adding 18px of height
     // to gameButtons. We want to get rid of that when we have don't have
     // instructions below game buttons
     marginBottom: -18
+  },
+  runImage: {
+    width: 26,
+    height: 26,
+    background: 'url("/blockly/media/common_images/shared-sprites-26x26.png") 0 0'
+  },
+  resetImage: {
+    width: 26,
+    height: 26,
+    background: 'url("/blockly/media/common_images/shared-sprites-26x26.png") 0 26px'
   }
 };
 
+export const RunButton = Radium(props => (
+  <button
+      id="runButton"
+      className={classNames(['launch', 'blocklyLaunch', props.hidden && 'invisible'])}
+      style={props.style}
+  >
+    <div>
+      {msg.runProgram()}
+    </div>
+    <img
+        src="/blockly/media/1x1.gif"
+        style={[!props.isMinecraft && styles.runImage]}
+        className={classNames([props.isMinecraft && "run26"])}
+    />
+  </button>
+));
+RunButton.propTypes = {
+  hidden: React.PropTypes.bool,
+  style: React.PropTypes.object,
+  // minecraft depends on styles sheets in some cases, instead of inlined
+  isMinecraft: React.PropTypes.bool
+};
+
+export const ResetButton = Radium(props => (
+  <button
+      id="resetButton"
+      className="launch blocklyLaunch"
+      style={[commonStyles.hidden, props.style]}
+  >
+    <div>
+      {msg.resetProgram()}
+    </div>
+    <img
+        src="/blockly/media/1x1.gif"
+        style={[!props.isMinecraft && styles.resetImage]}
+        className={classNames([props.isMinecraft && "reset26"])}
+    />
+  </button>
+));
```

Comment:
I will say functional stateless components feel nicer with ES6 where it saves us a level of indentation, i.e. otherwise I would have
```
var ResetButton = Radium(function (props) {
  return (
    <button>
    ....
    </button>
  );
});